torchdynamo/variables/torch.py

```diff
@@ -128,6 +129,8 @@ def call_function(
                 return ConstantVariable(True, **options)
             elif self.value is torch.is_floating_point:
                 return ConstantVariable(args[0].dtype.is_floating_point, **options)
+            elif self.value is torch.is_complex:
```

Comment:
Is this all we support, what about things like quantized `qint`?

Reply:
This pr is for getting past bugs and errors in bumping versions as per the title. If you want to request a torchdynamo feature please open a PR!